docker/README.md

```
@@ -62,6 +62,16 @@ docker run -d --init  --name signalk-server -p 3000:3000 -v $(pwd):/home/node/.s
 
 This will run the server as background process and current directory as the settings directory. You will be prompted to create admin credentials the first time you you access the configuration admin web UI.
 
+## Playback of a raw log file
+
+You can launch an ephemeral container named `signalk-playback` that disables all configured data connections and plays back a single raw log file instead. Mount the directory containing the log file and pass it to `--data`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)